### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs [hunk 1]
```diff
@@ -326,6 +326,9 @@ private static Process StartNewVisualStudioProcess(string installationPath, int
                 // Disable roaming settings to avoid interference from the online user profile
                 Process.Start(vsRegEditExeFile, $"set \"{installationPath}\" {Settings.Default.VsRootSuffix} HKCU \"ApplicationPrivateSettings\\Microsoft\\VisualStudio\" RoamingEnabled string \"1*System.Boolean*False\"").WaitForExit();
 
+                // Disable async completion for 16.0 Preview 3 testing
```
Comment (jaredpar) [inline src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs]: Do we have abug tracking turning this back on? Should reference it here

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs]: Enabling async completion in a public stable release is dependent on at least one successful integration test pass completing with it enabled, so we're conceptually tracking it at a high level.

File: eng/build.ps1 [hunk 2]
```diff
@@ -63,6 +63,33 @@ param (
 Set-StrictMode -version 2.0
 $ErrorActionPreference = "Stop"
 
+[Reflection.Assembly]::LoadWithPartialName("System.Drawing")
+[Reflection.Assembly]::LoadWithPartialName("System.Windows.Forms")
```
Comment (sharwell) [inline eng/build.ps1]: ❕ These need semicolons to avoid printing lines to the console when the script runs.

Comment (sharwell) [reply eng/build.ps1]: ➡️ Now using `Add-Type`

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs [hunk 3]
```diff
@@ -359,7 +359,7 @@ Inherits Attribute
 Import System;
 
 Public Class CustomAttribute
-        Inherits Attribute
+    Inherits Attribute
```
Comment (sharwell) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs]: @mavasani Please verify that the new behavior in this file is correct and not a regression

File: eng/build.ps1 [hunk 4]
```diff
@@ -505,6 +505,47 @@ try {
...
+                $sessionid = $quserItems[2]
+                if ($sessionid -eq 'Disc') {
+                    # When the session isn't connected, the third value is 'Disc' instead of the ID
+                    $sessionid = $quserItems[1]
+                }
+
+                if ($quserItems[1] -eq 'console') {
+                    Write-Host "Disconnecting from console before attempting reconnection"
+                    try {
+                        tsdiscon
```
Comment (jaredpar) [inline eng/build.ps1]: `Execute-Command tsdiscon`

File: eng/build.ps1 [hunk 5]
```diff
@@ -505,6 +505,47 @@ try {
...
+                    Start-Sleep -Seconds 3
+                    query user
+                }
+
+                Write-Host "tscon $sessionid /dest:console"
+                tscon $sessionid /dest:console
+
+                # Connection is asynchronous, so wait a few seconds for it to complete
+                Start-Sleep 3
+                query user
```
Comment (jaredpar) [inline eng/build.ps1]: This is an EXE whose failures will be ignored. Is that intended?

Comment (sharwell) [reply eng/build.ps1]: Yes, it just shows the output in the build log.

File: eng/build.ps1 (hunk 4 above)
Comment (sharwell) [reply eng/build.ps1]: What's the difference here?

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs (hunk 3 above)
Comment (mavasani) [reply src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs]: Seems like completion is changing the formatting for the next line. I believe the previous behavior is more desirable, which seems to be the case behavior in presence of async completion.

File: src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs (hunk 1 above)
Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs]: ➡️ Filed #33797

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs (hunk 3 above)
Comment (sharwell) [reply src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicRename.cs]: ➡️ Filed #33799

File: eng/build.ps1 (hunk 4 above)
Comment (sharwell) [reply eng/build.ps1]: ➡️ Follow-up pull request #33800 submitted

File: eng/build.ps1 (hunk 5 above)
Comment (sharwell) [reply eng/build.ps1]: ➡️ Follow-up pull request #33800 submitted

**Pull request merged**